build.gradle.kts

```diff
@@ -204,9 +257,21 @@ spotless {
                 }
         )
         googleJavaFormat()
+        licenseHeader(headerWithStars).yearSeparator(" - ")
     }
     kotlin {
         ktfmt().kotlinlangStyle()
+        licenseHeader(headerWithStars).yearSeparator(" - ")
+    }
+
+    python {
+        target("src/main/**/*.py")
+        licenseHeader(headerWithHashes, "from").yearSeparator(" - ")
+    }
+
+    format("golang") {
```

Comment:
no license header for test files?

Reply:
Well there aren't any :) But I can add the glob for test files for futureproofness.

Reply:
src/test/resources/golang ?

Reply:
I'm skipping it for the test files for now since this creates problems with the test themselves.

Reply:
> src/test/resources/golang ?

Those are only files that are being tested. I would argue that those snippets do not need a license header